### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by ikoote1**
Changes made:
- Added all components required.
- Styled the application based on the design.
- Pushed the code and open a pull request.

Thanks for reviewing my work.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -44,7 +44,34 @@
...
+
+.chart {
+  width: 92%;
+  position: relative;
+  left: 3%;
+  border-radius: 10px;
+  height: 2px;
+  background-color: rgb(255, 255, 255);
+}
+
```
Comment (tresorsawasawa) [inline src/index.css]: - [x] As  required for the project, kindly make sure your styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) e.g.: Text colors, Fonts(size, weight, family,...), background-colors, alignment, spscing(psdding & mergin),...
### Template design
Screenshot from 2023-04-06 19-22-04

### Your Design
Screenshot from 2023-04-06 19-27-39

File: src/components/navigation.js [hunk 2]
```diff
@@ -12,6 +13,7 @@ const Navigation = () => (
           <Link style={{ textDecoration: 'none', color: 'black' }} to="categories">CATEGORIES</Link>
         </li>
       </ul>
+      <span><IoIosPerson className="profile" /></span>
     </nav>
   </div>
```
Comment (tresorsawasawa) [inline src/components/navigation.js]: - [x] As  required for the project, kindly make sure your styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) e.g.: Text colors, Fonts(size, weight, family,...), background-colors, alignment, spscing(psdding & mergin),...
### Template design
Screenshot from 2023-04-06 19-33-41

### Your Design
Screenshot from 2023-04-06 19-34-00

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @ikoote1 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js [hunk 3]
```diff
@@ -1,37 +1,65 @@
...
+                  <div className="update">
+                    <h3>CURRENT CHAPTER</h3>
+                    <h2>Introduction</h2>
+                    <button type="button">UPDATE PROGRESS</button>
+                  </div>
+                </div>
+              </div>
+            ),
+          )}
+          <div className="chart" />
```
Comment (nahnahdev) [inline src/components/Books.js:58]: - Great job so far @ikoote1 Your design just needs a little fix and that will be all :+1:
- [x] I see that you made an effort to implement the previous reviewer's suggestion. However, the right side of your design isn't properly styled as the previous reviewer pointed out, and aside from that,
- [x] There is a horizontal scroll effect on your page that will make navigating through your page an unpleasant experience for the end user so kindly fix that as well. I have attached a video reference below :+1:   

- Video Reference 

[Screencast from 04-07-2023 10:42:22 AM.webm](https://user-images.githubusercontent.com/102699958/230597439-e379d77f-3d15-4615-80c7-cd0218c4eeab.webm)

- [x] Please ensure that your final design is well centered on the page as shown in the Zeplin design as the previous reviewer mentioned :+1: 
- Given Design
image

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @ikoote1 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.


 ## Highlights
- No linter errors :heavy_check_mark: 
- Professional README :heavy_check_mark: 
- Good use of GitFlow :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/navigation.js [hunk 4]
```diff
@@ -1,17 +1,19 @@
 import { Link } from 'react-router-dom';
+import { IoIosPerson } from 'react-icons/io';
 
 const Navigation = () => (
   <div className="bar">
     <nav className="navbar">
-      <h1>Bookstore Ikoote</h1>
+      <h1>Bookstore</h1>
```
Comment (nahnahdev) [inline src/components/navigation.js]: - [x] Kindly make this `Bookstore CMS` as seen in the design :wink: So your design looks exactly like the given design :100:

**Review by DeliceLydia — APPROVED** [review]
## Hi @ikoote1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- Addressed all changes requested on the design


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**